modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/topology/api/LogicalNode.java

```diff
@@ -95,7 +95,8 @@ public LogicalNode(
 
         this.userAttributes = userAttributes == null ? Collections.emptyMap() : userAttributes;
         this.systemAttributes = systemAttributes == null ? Collections.emptyMap() : systemAttributes;
-        this.storageProfiles = storageProfiles == null || storageProfiles.isEmpty() ? List.of("dummy") : storageProfiles;
+        // TODO: https://issues.apache.org/jira/browse/IGNITE-21388 remove default
+        this.storageProfiles = storageProfiles == null || storageProfiles.isEmpty() ? List.of("default") : storageProfiles;
```

Comment:
do you have ideas how this will be fixed? what if user doesn't set any storage profile?

Reply:
The node will not be a part of any zone, I think.

Reply:
do we plan to have some default storage profile for a node? won't this removal ruin all tests, because nodes do not have storage profile set explicitly?

Reply:
Yep, this code https://github.com/gridgain/apache-ignite-3/pull/92/files#diff-d9d730665f2779103d4596b8d18d997652b05dadf151a8154127598c68a73e9dR81 do this trick.

The page memory module patch the configs and adds the default storage profile with aipersist engine. So, by default we will have:
- "default" storage profile in node configs
- "default" storage profiles in the default zone
- "default" storage profile in the table from the default zone
And the whole flow will be working out of the box on the fresh installation.

Reply:
thanks! this shed a bit of light! However, I see that javadoc of `patchConfigurationWithDynamicDefaults` says that "This method is called only for cluster-wide configuration on cluster initialization", but storage profile is a local config. This fact makes me a bit confused.

Reply:
Answered here https://github.com/gridgain/apache-ignite-3/pull/92#discussion_r1478715938